keadm/Makefile

```diff
@@ -1,16 +1,16 @@
 
-# make kubeedge
-.PHONY: default kubeedge
-kubeedge:
-	go build -o kubeedge keadm.go
+# make keadm
+.PHONY: default keadm
+keadm:
+	go build -o keadm keadm.go
 
 lint:
 	golangci-lint run --disable-all -E golint ./...
 	go vet ./...
 
 .PHONY: clean
 clean:
-	rm kubeedge
+	rm keadm
```

Comment:
please use $(RM) instead of rm; 

$(RM) doesn't throw an error if the file doesn't exists